test/test_nn.py

```diff
@@ -10007,31 +10007,41 @@ def test_upsamplingBicubic2d_aa_correctness(self, device, memory_format):
         self.assertEqual(expected_out, t_out)
 
     @parametrize_test("align_corners", [True, False])
-    def test_upsamplingTrilinear3d(self, device, align_corners):
+    @parametrize_test("memory_format", [torch.contiguous_format, torch.channels_last_3d])
+    def test_upsamplingTrilinear3d(self, device, align_corners, memory_format):
```

Comment:
Don't we have an opinfo for this?

Reply:
Yes, there is one, but it does not test the failed case. Also `test_upsamplingTrilinear3d` already existing, I just put memory_format into parametrization and added grads consistency check between CF and CL which testing the fix proposed in this PR

Reply:
It'd be nice to add the relevant failing samples to the OpInfo, that way we would get proper testing for these inputs.

Reply:
Actually, it is not about specific sample but a specific way of testing. We have to ensure that grad_input is consistent for grad_output contig vs grad_output channels_last3d. Probably, there is no such test in test_ops.py and gradcheck does not do that...

Reply:
we do have a non contiguous test, I think that would exercise this path, wouldn't it?

Reply:
ah, no, this is for the backward. Ok, let's leave it as is

Reply:
hmmm, the non contig test does exercise the backward as well tho... oh well